### ISSUE TITLE
How to prevent overlapping segments?

### ISSUE THREAD

**Issue by eyildiz-ugoe** [matterport/Mask_RCNN#1747]
The segmentation map delivered by the algorithm has overlapping segments as you can see in the below image, which is not desired. How to make sure that the segments don't overlap, and that each pixel belongs to only one class? There has to be an option for that, right?

![segmentation_map](https://user-images.githubusercontent.com/35396058/65134789-2ad7fb00-da05-11e9-87b9-641956278b3a.jpg)

**Comment by zlyx525** [comment]
I have the same problem. My result is like this. Have you solved it? 
![image](https://user-images.githubusercontent.com/29477518/65224478-d6607880-daf5-11e9-8276-9dee1b750978.png)

**Comment by eyildiz-ugoe** [comment]
Unfortunately not yet. I do believe other people have this problem too, so we may keep looking how they solved it. In any case, if I find a solution, I'll post here. 

I also hoped the authors to say something about this issue.

**Comment by cipher009** [comment]
Any news on that?

**Comment by eyildiz-ugoe** [comment]
Not really, besides, I am not using the aforementioned network anymore.
